### DISCUSSION TITLE
专业节点搭建，定制APP，定制平台系统开发

### DISCUSSION THREAD

**Discussion by runtuer** [filecoin-project/community-china#197] [应用开发]
IPFS圈子
FILECOIN节点搭建、封装，平台管理系统出售
APP分币平台，多模式定制开发
支持包括不限于：
三级分销、拼团，抢购等
![如需了解详情，请加V， 请备注来意] ![avatar](http://test.fenbi.xjcc.top/assets/index/images/theseaer-wx.jpg)

**Comment by Chenhongwe** [comment]
Fil算力填充，240/T，包半年运维，送七牛商业存储软件一年，坐标杭州，欢迎骚扰 phone：15255579473（同V）

**Comment by zhaobo443** [comment]
小型机房建设，专业IPFS节点搭建，分币系统app定制全套filecoin,包括测ceph存储技术，长期合作全部免费奉送，长期维护只抽取5%服务费！电话：19965407481
